lib/ansible/module_utils/six/_six.py

```diff
@@ -215,7 +215,7 @@ def is_package(self, fullname):
         We need this method to get correct spec objects with
         Python 3.4 (see PEP451)
         """
-        return hasattr(self.__get_module(fullname), "__path__")
+        return getattr(self.__get_module(fullname), "__path__", None) is not None
```

Comment:
This file should not be modified as it is a censored copy of upstream six.  This change would cause our copy to diverge